cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -76,6 +76,40 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+#. Set the YARN Application Classpath.
+
+   .. highlight:: console
+
+   If you are running Spark programs, CDAP requires that additional entries be added to
```

Comment:
If you plan run Spark programs from CDAP. 

Because the user can just run spark jobs outside CDAP without doing this.